evap/contributor/templates/contributor_evaluation_form.html

```diff
@@ -21,11 +23,11 @@
             {% blocktrans %}You cannot edit this evaluation because it has already been approved.{% endblocktrans %}
         {% endif %}
     </div>
-    <h3>
-        {{ evaluation.full_name }} ({{ evaluation.course.semester.name }})
-    </h3>
+    <h3>{{ evaluation.full_name }} ({{ evaluation.course.semester.name }})</h3>
 
-    <form method="POST" class="form-horizontal multiselect-form tomselectform" id="evaluation-form">
+    <form method="POST"
+          class="form-horizontal multiselect-form tomselectform"
+          id="evaluation-form">
```

Comment:
The indentation of attributes is dependent on the tag length. here it is 6 spaces, for a button it would be 8 spaces.

Reply:
this looks interesting :thinking: ; why can't everyone just agree that prettier does it best (tm):
```html
<form
    method="POST"
    class="form-horizontal multiselect-form tomselectform"
    id="evaluation-form"
>
    I am a form
</form>
```